libr/bin/p/bin_elf.inc

```diff
@@ -57,6 +57,17 @@ static void setimpord(ELFOBJ* eobj, ut32 ord, RBinImport *ptr) {
 	eobj->imports_by_ord[ord] = r_bin_import_clone (ptr);
 }
 
+static char *setphname(ut16 mach, Elf_(Word) ptyp) {
+	// TODO to complete over time
+	if (mach == EM_ARM) {
+		if (ptyp == SHT_ARM_EXIDX) {
+			return strdup ("EIXDX");
```

Comment:
We should evaluate the use of r_str_const to save some memory in here. But we can evolve the concept with a stringpool. Nothing to worry in this pr. But just to think about